### DISCUSSION TITLE
matchUpdateTypes clarification

### DISCUSSION THREAD

**Discussion by adam-moss** [renovatebot/renovate#18342] [Request Help] [answered]
Hi,

Under the config docs for `matchUpdateTypes` is states the allowed values are:

> major, minor, patch, pin, pinDigest, digest, lockFileMaintenance, rollback, bump

Apologies if it is documented already (my search foo is failing me), but what would trigger a type of `lockFileMaintenance`, `rollback`, or `bump` ?

Thanks!

**Comment by rarkins** [comment, marked as answer]
lockFileMaintenance - happens if lockFileMaintenance.enabled=true in config. 

rollback - happens if rollbackPrs=true 

bump - happens if rangeStrategy=bump
